src/services/infra/StatsService.ts

```diff
@@ -105,6 +106,15 @@ export class StatsService {
       version: Versions.V2,
     })
   }
+
+  incrementGithubApiCall = (method: Method, site: string) => {
+    this.statsD.increment("users.github.api", {
+      site,
```

Comment:
So on DD we will be able to filter by site to githubApiCallsMade ?

Reply:
yes - i've an `if` statement just in case of billing cos there was a prev incident before